x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -99,7 +100,9 @@ const UnstyledProcessEventDot = React.memo(
      */
     timeAtRender: number;
   }) => {
-    const resolverComponentInstanceID = useSelector(selectors.resolverComponentInstanceID);
+    const resolverComponentInstanceID = useSelector((state: ResolverState) =>
```

Comment:
For my own edification, what's the difference with using the arrow function vs just passing in `selectors.resolverComponentInstanceID`? Does it not pass the `state` by default?